.github/workflows/ci.yml

```diff
@@ -70,6 +71,33 @@ jobs:
       - name: Test
         run: cargo test --all --exclude niri-visual-tests ${{ matrix.release-flag }} -- --nocapture
 
+  build-musl:
+    strategy:
+      fail-fast: false
+
+      matrix:
+        configuration: [release]
```

Comment:
Maybe should just build in debug? Should be reasonably faster, and I don't think we diverge from release. The Fedora CI also builds in debug.